src/test/incremental/dirty_clean.rs

```diff
@@ -27,14 +27,14 @@ mod y {
 
     #[rustc_clean(label="typeck_tables_of", cfg="cfail2")]
     pub fn y() {
-        //[cfail2]~^ ERROR `typeck_tables_of(y::y)` should be clean but is not
+        //[cfail2]~^ ERROR `typeck_tables_of(y)` should be clean but is not
         x::x();
     }
 }
 
 mod z {
     #[rustc_dirty(label="typeck_tables_of", cfg="cfail2")]
     pub fn z() {
-        //[cfail2]~^ ERROR `typeck_tables_of(z::z)` should be dirty but is not
+        //[cfail2]~^ ERROR `typeck_tables_of(z)` should be dirty but is not
```

Comment:
These don't seem unambiguous, how does this happen? Maybe we should ignore namespaces, to be conservative?
(Although presumably this has been discussed already)

Reply:
So it looks like function name `z` and module name `z` are not colliding on the disambiguation check. If they were, then `z` name would have been excluded from path shortening. Not sure why yet. Functions are in `ValueNS` AFAIK, and modules - where? Maybe under `TypeNS` with `DefKind::Mod`? I'll take a closer look.

Reply:
This is fixed as of a few days ago.